rllib/core/rl_trainer/tests/test_trainer_runner.py

```diff
@@ -55,7 +55,7 @@ def test_trainer_runner_local(self):
             print(f"Testing framework: {fw}, scaling mode: {scaling_mode}")
             ray.init(ignore_reinit_error=True)
             scaling_config = self.scaling_configs[scaling_mode]
-            runner = get_trainer_runner(fw, env, scaling_config)
+            runner = get_trainer_runner(fw, env, scaling_config, eager_tracing=True)
```

Comment:
This is ok, since eager_tracing = True is a superset of the behavior when eager_tracing = False. + it runs faster.